app/views/spotlight/search_configurations/_facet_metadata.html.erb

```diff
@@ -6,9 +6,4 @@
   <% else %>
     <%= content_tag :span, t(:'.value_count', count: metadata[:value_count]) %>
   <% end %>
-  <% if metadata[:terms].any? %>
```

Comment:
I'm guessing if we're removing this, there's some controller or model-land code that can also be thrown out?

Reply:
Yeah, I assume we'll need it again down the road for the issue that was spun out, but w/o knowing the data that we want to present, it can probably be safely removed and re-added later to fit our needs.